docs/contents/components/disclosure/accordion/accessibility.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Accordion
+description: "`Accordion` is a component of a list that unfolds and collapses information."
+package_name: "@yamada-ui/accordion"
+with_description: true
+tab: Accessibility
+---
+
+`Accordion` follows the [WAI-ARIA - Accordion Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) for accessibility.
+
+## Keyboard Navigation
+
+| key              | Description                                                                                                                   | Condition                              |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- |
+| `Tab`            | When focus moves to the accordion, it moves to the first item. <br/> When within the accordion, it moves to the next item.    | -                                      |
+| `Shift` + `Tab`  | When focus moves to the accordion, it moves to the last item. <br/> When within the accordion, it moves to the previous item. | -                                      |
```

Comment:
it might be better to delete `When focus moves to the accordion, it moves to the last item. <br/>`.